config/crd/kustomization.yaml

```diff
@@ -18,21 +18,6 @@ patches:
       name: lvmclusters.lvm.topolvm.io
     path: patches/additionalPrinterColumn_in_lvmcluster.yaml
 
-patchesStrategicMerge:
```

Comment:
I just want to mention here that removing these parts will force us to never work outside an openshift cluster with certificates ever. People usually need cert manager patches outside otherwise the webhooks will not work.

Reply:
Clarified: can be removed